cmd/server/handler.go

```diff
@@ -208,6 +213,13 @@ func serve(c *config.Config, cmd *cobra.Command, handler http.Handler, address s
 		},
 	})
 
+	srv.RegisterOnShutdown(func() {
+		tracer := c.GetTracer()
+		if tracer.IsLoaded() {
```

Comment:
Same here, or depending on what `Close()` returns (if it returns nothing) maybe:

```
if tracer := c.GetTracer(); tracer.Loaded() {
  srv.RegisterOnShutdown(tracer.Close)
}
```